src/generateCode.js

```diff
@@ -30,10 +30,12 @@ export function areaToCSS(area, { parentGrid, useTemplateAreas = true, validTemp
 }
 
 export function gridToCSS(name, grid, { useTemplateAreas = true, repeat }) {
-  let css = `  display: grid;
-  grid-template-columns: ${namedTemplateColumns(grid, repeat)};
-  grid-template-rows: ${namedTemplateRows(grid, repeat)};
-  gap: ${grid.row.gap + ' ' + grid.col.gap};` // TODO: cssGridGap(grid)
+  let css = `
+    display: grid;
+    grid-template-columns: ${namedTemplateColumns(grid, repeat)};
+    grid-template-rows: ${namedTemplateRows(grid, repeat)};
+    gap: ${grid.row.gap} ${grid.col.gap};
+  ` // TODO: cssGridGap(grid)
```

Comment:
I like this change, but please remove the added newline in line 33, and the one at the end of line 37, or the generated code in CodePen will have extra lines there (this code is only used when exporting to CodePen at the moment)

Reply:
Done